### PR TITLE
test(MM-T884): Updating existing and adding step 3 to the test case

### PR DESCRIPTION
**Pull request by jgilliam17**
#### Summary

MM-T884 -  Updating existing steps and adding new step 3 

#### Is the feature in stable branch or under review?
- [] Feature in stable branch (master, main)?
- [x] Feature still under review? If yes, list all related pull requests for reference.
Related PRs:
https://github.com/mattermost/mattermost-server/pull/21639
https://github.com/mattermost/mattermost-webapp/pull/11716
#### Test server

Changes can be seen on this server before merge to main https://mattermost-server-pr-21639.test.mattermost.cloud/
#### Test client

- [x] Browser
- [] Mobile App (version, OS)
- [] Desktop App (version, OS)

<a href="https://gitpod.io/#https://github.com/mattermost/mattermost-test-management/pull/39"></a>

### CODE REVIEW

File: data/test-cases/channels/channel-settings/MM-T884.md [hunk 1]
```diff
@@ -46,8 +46,7 @@ steps_hashed: 2866e48395e4a24f42328476be1bb81aed576f28f6ae3677527fd16bdc2a7d71e0
 
 **Step 1**
 
-Renaming channel name validates against two user IDs being used in URL\
-–––––––––––––––––––––––––\\
+Renaming channel name validates against two user IDs being used in URL
 
 1. Create a new channel with a URL that contains `\_\_` (**2 underscores**) in it
```
Comment (mattermost-quality) [inline data/test-cases/channels/channel-settings/MM-T884.md:51]: This renders with the forward slash. I would suggest to change as:
```suggestion
1. Create a new channel with a URL that contains "__" (**2 underscores**) in it
```

File: data/test-cases/channels/channel-settings/MM-T884.md [hunk 2]
```diff
@@ -70,19 +69,30 @@ _Important notes about this step:_
...
+
+---
+
+**Step 3**
+
+Creating a new channel validates against gm-like names being used as channel name
+
+1. Click on create public channel
+   1. Verify that the new channel modal is visible
+3. Add the new channel name with invalid name and press Create Channel e.g. 71b03afcbb2d503d49f87f057549c43db4e19f92
```
Comment (mattermost-quality) [inline data/test-cases/channels/channel-settings/MM-T884.md:92]:
```suggestion
2. Add the new channel name with invalid name and press Create Channel e.g. 71b03afcbb2d503d49f87f057549c43db4e19f92
```

File: data/test-cases/channels/channel-settings/MM-T884.md [hunk 3]
```diff
@@ -70,19 +69,30 @@ _Important notes about this step:_
...
+   1. Assert the error occured with the appropriate message " Channel names can\'t be in a hexadecimal format. Please enter a different channel name."
```
Comment (mattermost-quality) [inline data/test-cases/channels/channel-settings/MM-T884.md:93]:
```suggestion
   1. Assert the error occurred with the appropriate message " Channel names can\'t be in a hexadecimal format. Please enter a different channel name."
```

File: data/test-cases/channels/channel-settings/MM-T884.md [hunk 4]
```diff
@@ -70,19 +69,30 @@ _Important notes about this step:_
...
+5. Close the create channel modal
```
Comment (mattermost-quality) [inline data/test-cases/channels/channel-settings/MM-T884.md:94]:
```suggestion
3. Close the create channel modal
```

File: data/test-cases/channels/channel-settings/MM-T884.md [hunk 5]
```diff
@@ -70,19 +69,30 @@ _Important notes about this step:_
 
 - Channel url is invalid and that the relevant error message is returned
 
-![](https://smartbear-tm4j-prod-us-west-2-attachment-rich-text.s3.us-west-2.amazonaws.com/embedded-f3277290f945470c4add5d21ef3dc7ca7b74388fc7152bfb6b99ae58c66a95a8-1589215826306-1589215826306.png)
-
 ---
 
 **Step 2**
 
 Creating new channel validates against two user IDs being used as channel name\
```
Comment (mattermost-quality) [inline data/test-cases/channels/channel-settings/MM-T884.md:76]:
```suggestion
Creating new channel validates against two user IDs being used as channel name
```

File: data/test-cases/channels/channel-settings/MM-T884.md [hunk 6]
```diff
@@ -70,19 +69,30 @@ _Important notes about this step:_
...
-\--------------------------\\
 
 1. Attempt to create a channel with the channel name: `uzsfmtmniifsjgesce4u7yznyh\_\_uzsfmtmniifsjgesce5u7yznyh`
```
Comment (mattermost-quality) [inline data/test-cases/channels/channel-settings/MM-T884.md:78]:
```suggestion
1. Attempt to create a channel with the channel name: `uzsfmtmniifsjgesce4u7yznyh__uzsfmtmniifsjgesce5u7yznyh`
```

**Review by mattermost-quality — CHANGES_REQUESTED** [review]
.

File: data/test-cases/channels/channel-settings/MM-T884.md (hunk 1 above)
Comment (saturninoabril) [inline data/test-cases/channels/channel-settings/MM-T884.md:51]: nit: remove forward slashes (used for escaping) to render nicely.
```suggestion
1. Create a new channel with a URL that contains "__" (**2 underscores**) in it
```

File: data/test-cases/channels/channel-settings/MM-T884.md (hunk 5 above)
Comment (saturninoabril) [inline data/test-cases/channels/channel-settings/MM-T884.md:76]:
```suggestion
Creating new channel validates against two user IDs being used as channel name
```

File: data/test-cases/channels/channel-settings/MM-T884.md (hunk 6 above)
Comment (saturninoabril) [inline data/test-cases/channels/channel-settings/MM-T884.md:78]: Here and do the same in line 53 (to be closed with backticks), 59 and 60 (to be closed with backticks).
```suggestion
1. Attempt to create a channel with the channel name: `uzsfmtmniifsjgesce4u7yznyh__uzsfmtmniifsjgesce5u7yznyh`
```

File: data/test-cases/channels/channel-settings/MM-T884.md (hunk 2 above)
Comment (saturninoabril) [inline data/test-cases/channels/channel-settings/MM-T884.md:92]:
```suggestion
2. Add the new channel name with invalid name and press Create Channel e.g. 71b03afcbb2d503d49f87f057549c43db4e19f92
```

File: data/test-cases/channels/channel-settings/MM-T884.md (hunk 3 above)
Comment (saturninoabril) [inline data/test-cases/channels/channel-settings/MM-T884.md:93]:
```suggestion
   1. Assert the error occurred with the appropriate message " Channel names can\'t be in a hexadecimal format. Please enter a different channel name."
```

File: data/test-cases/channels/channel-settings/MM-T884.md (hunk 4 above)
Comment (saturninoabril) [inline data/test-cases/channels/channel-settings/MM-T884.md:94]:
```suggestion
3. Close the create channel modal
```

**Review by saturninoabril — CHANGES_REQUESTED** [review]
Thanks Jelena! Please see comment/suggestions.

**Review by saturninoabril — APPROVED** [review]
Thanks Jelena!

**Pull request merged**